azure-pipelines/mypy_test_jobs.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
```

Comment:
Thanks for the change.

I think that this is a bit of an over-engineering.

For mypy the "template" should be use only as an `include` semantic and not as a generator.

I feel that splitting the rules over 2 files, and defining the Ubuntu image name as a variable is only making it harder to understand what is going on there.

Below is my suggestions for the content of this file, and the other file can be removed.

Also, since this is only one job, I think that is best to name this file just `mypy.job.yml` or something like that.

This is all about `testing` so using `test` in the name doesn't add much value... and this is a single job ...so `jobs` is kind of misleading.

I would prefer to see the test in red, as failing... so that we can see the actual result.

It should not be marked as required, so it should not block any merge.

But it will serve as a reminder that mypy is not really green.

```
jobs:
- job: 'ubuntu_mypy'
  displayName: 'Mypy Ubuntu'
  pool:
    vmImage: 'ubuntu-latest'
  steps:
  - task: UsePythonVersion@0
    displayName: "Use Python 3.8"
    inputs:
      versionSpec: "3.8"

  - script: |
     python --version
     python -c "import sys; print(sys.prefix)"
     python -c "import sys; print(sys.exec_prefix)"
     python -c "import sys; print(sys.executable)"
     python -c "import struct; print(struct.calcsize('P') * 8)"
    displayName: 'Get Python Information'

  - script: 'python -m pip install -U pip setuptools tox'
    displayName: 'Update pip & install tox'

  - script: 'python -m tox -e mypy'
    displayName: 'Run mypy'

```